manifests/fpm/config.pp

```diff
@@ -103,12 +103,18 @@
     default => $log_group,
   }
 
+  $validate_cmd = $php::fpm::disable_configtest ? {
```

Comment:
Why wouldn't you want to validate this? Not sure the boolean is needed.

Reply:
In my use case, I want validation always. But, I took the idea from [auditbeat's module](https://forge.puppet.com/norisnetwork/auditbeat), which gives the opportunity to maintain the behaviour as today, being the default value that enables validation.